app/src/main/java/com/google/samples/apps/topeka/helper/TransitionHelper.java

```diff
@@ -23,16 +23,21 @@
      * @return All transition participants.
      */
     public static Pair[] createSafeTransitionParticipants(@NonNull Activity activity,
-            @Nullable Pair... otherParticipants) {
+            boolean excludeStatusBar, @Nullable Pair... otherParticipants) {
         // Avoid system UI glitches as described here:
         // https://plus.google.com/+AlexLockwood/posts/RPtwZ5nNebb
         View decor = activity.getWindow().getDecorView();
-        View statusBar = decor.findViewById(android.R.id.statusBarBackground);
+        View statusBar = null;
+        if (!excludeStatusBar) {
+            statusBar = decor.findViewById(android.R.id.statusBarBackground);
+        }
         View navBar = decor.findViewById(android.R.id.navigationBarBackground);
 
         // Create pair of transition participants.
         List<Pair> participants = new ArrayList<>(3);
-        addNonNullViewToTransitionParticipants(statusBar, participants);
+        if (!excludeStatusBar) {
```

Comment:
This check is redundant as the method call to `addNonNullViewToTransitionParticipants` already checks for null values.